### ISSUE TITLE
Video stream does not resume after stopping

### ISSUE THREAD

**Issue by justinhsg** [smartdevicelink/sdl_java_suite#1102]
### Bug Report
We are trying to use video streaming on Android, and we found that we cannot resume video streaming after calling stopStreaming()

##### Reproduction Steps
1. Start SdlService, and set up for video streaming
2. Call the method sdlManager.getVideoStreamManager().startRemoteDisplayStream(). The video stream should start
3. Call sdlManager.getVideoStreamManager().stopStreaming()
4. Resume the stream via sdlManager.getVideoStreamManager().resumeStreaming()

##### Expected Behavior
The video stream should resume

##### Observed Behavior
The video stream does not resume after stopping

##### OS & Version Information
* Android Version: 7.0
* SDL Android Version: 4.9.0
* Testing Against: Tested with SDL Core version 5.1.3, tested both over USB and TCP connection

##### Additional Information
I have an idea of what might be causing this issue (StreamingStateMachine), but I am not completely certain; I'll add it as a comment to this issue.

**Comment by justinhsg** [comment]
When checking out the state of StreamingStateMachine via currentVideoStreamState(), the method always returns 0 (StreamingStateMachine.NONE).

In VideoStreamManager.java, there doesn't seem to be a transition from StreamingStateMachine.NONE to StreamingStateMachine.READY

Hence when resumeStreaming() is called:
```
public void resumeStreaming(){
	if(stateMachine.getState() != StreamingStateMachine.STOPPED){
		return;
	}
	startEncoder();
}
```
The method always returns without calling startEncoder()

**Comment by BrettyWhite** [comment]
Closed with #1128
